municipal_finance/static/javascript/table.js

```diff
@@ -800,7 +816,23 @@
                 var data = muni_data ? muni_data[functions[f].code] : null;
                 this.renderMuniValues(muni, data, tr);
               }
-            } else {
+            }
+            else if (cube.model.can_aggregate) {
```

Comment:
I think let's use `if cube.name == 'capital_v2'` here instead.

The need for an additional row of column headings is a bit more complex than simply that _some_ column in the cube can not be aggregated. So let's not do logic based on can_aggregate yet.

Reply:
changed to `if (CUBE_NAME == 'capital_v2')`